src/train_tracker.py

```diff
@@ -0,0 +1,228 @@
+"""列車捕捉モジュール.
+
+列車を捕捉する
+@author: desty505 miyashita64
+"""
+
+import cv2
+import numpy as np
+from camera_interface import CameraInterface
+
+
+class TrainTracker:
+    """列車を捕捉するクラス."""
+
+    def __init__(self, camera_id: int) -> None:
+        """コンストラクタ.
+
+        Args:
+            camera_id (int): カメラID
+        """
+        self.camera = CameraInterface(camera_id)
+        self.diff_border = 20
```

Comment:
これは閾値？
```suggestion
        self.diff_border = 20 #閾値
```
もしくは
```suggestion
        self.threshold= 20
```
わかりにくいのでコメントか英語にお願いします